### PR TITLE
Soc instructions

### PR DESCRIPTION
**Pull request by moy**
Too late for this year, but we have so many wrong applications, let's try to clarify them now so that we have a better template next year. What do you think?

(we can merge this right after the deadline)

### CODE REVIEW

**Comment by peff** [comment]
Thanks, I think all of the changes here make sense. :+1:

**Comment by moy** [comment]
The deadline was friday. I added commits by @chriscool. Everybody's OK?

File: SoC-2016-Microprojects.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ libgit2 list of
 projects](https://github.com/libgit2/libgit2/blob/master/PROJECTS.md)
 rather than the list below.*
 
+Git development is based on sending successive versions of patchs or
```
Comment (peff) [inline SoC-2016-Microprojects.md]: Usually we'd say `patches` in English (I thought initially it was just a typo, but TIL that it's `patchs` in French).

File: SoC-2016-Microprojects.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ libgit2 list of
...
+patch series to the mailing list until they are considered good and
+correct by the reviewers and Junio Hamano, the maintainer, who will
+merge them. This process usually takes quite some time. By sending
+drafts of your patches for your microproject, to the
```
Comment (peff) [inline SoC-2016-Microprojects.md]: I think this comma is extraneous. Might also be easier to parse as `sending drafts of your microproject patches to the mailing list...` to reduce the recursion level of prepositional phrases.

File: SoC-2016-Microprojects.md [hunk 3]
```diff
@@ -19,6 +19,18 @@ libgit2 list of
...
+mailing list long before the deadline, you can show us that you are
+willing and able to work well using the Git development process.
+
+It is *expected* that what you send will need several rounds of
+reviews and discussions. If you not sure at all about a patch you can
```
Comment (peff) [inline SoC-2016-Microprojects.md]: `s/you not/you are not/`

**Comment by peff** [comment]
I pointed out a few minor issues, but other than that, feel free to merge.

By the way, you may want to just start pushing to non-master branches directly in the git/git.github.io repo, and opening PRs from there. If you do, then other collaborators on the project can push onto the branch to update the PR, too (and I probably would have done so for minor things like typo fixups).

Of course, if you'd prefer people not push onto your branches, then don't. :)

**Comment by moy** [comment]
Applied your remarks. Indeed, pushing to non-master branch in this repo would be better.

Thanks,

**Pull request merged**